dist/sordid-dragon.jquery.js

```diff
@@ -2,7 +2,7 @@
 // Copyright Poll Everywhere
 // Paul Cortens, Mike Foley and Adam Heath
 // https://github.com/polleverywhere/sordid-dragon
-// Version 2.0.0
+// Version 2.0.1
```

Comment:
Something went wrong with the version number when this was generated. It should be `2.1.0`.